integration-tests/container/perf/tools/do_syscall_64.py

```diff
@@ -47,9 +52,15 @@ def exit_handler(bpf, output, *args):
         # plain Exception so it is not possible to be more refined.
         print(f"[*] detaching failed: {e}")
 
+    print(f"possibly lost {g_lost_count} events")
```

Comment:
I may be missing this, but do we extract the logs for the container running this script as part of the artifacts?

Reply:
yeah, this is capture and will end up in the integration tests log, so should be available.